yarn.lock

```diff
@@ -12051,6 +12051,11 @@ uuid@^3.0.1, uuid@^3.3.2:
   resolved "https://registry.yarnpkg.com/uuid/-/uuid-3.4.0.tgz#b23e4358afa8a202fe7a100af1f5f883f02007ee"
   integrity sha512-HjSDRw6gZE5JMggctHBcjVak08+KEVhSIiDzFnT9S9aegmp85S/bReBVTb4QTFaRNptJ9kuYaNhnbNEOkbKb/A==
 
+uuid@^8.3.2:
```

Comment:
why didn't it remove the old one?

Reply:
@10xLaCroixDrinker it is used by `request@^2.88.0:` apparently

Reply:
It uses `uuid "^3.3.2"`